packages/website/ts/components/wallet/wrap_ether_item.tsx

```diff
@@ -0,0 +1,185 @@
+        borderBottomLeftRadius: 3,
+        borderTopRightRadius: 3,
+        borderTopLeftRadius: 3,
+        padding: 4,
+        width: 125,
+    },
+    ethAmountInput: { height: 32 },
+    innerDiv: { paddingLeft: 60, paddingTop: 0 },
+    wrapEtherConfirmationButtonContainer: { width: 128, top: 16 },
+    wrapEtherConfirmationButtonLabel: {
+        fontSize: 10,
+        color: colors.white,
+    },
+};
+
+export class WrapEtherItem extends React.Component<WrapEtherItemProps, WrapEtherItemState> {
+    constructor(props: WrapEtherItemProps) {
+        super(props);
+        this.state = {
+            currentInputAmount: undefined,
+            currentInputHasErrors: false,
+            isEthConversionHappening: false,
+        };
+    }
+    public render() {
+        const etherBalanceInEth = ZeroEx.toUnitAmount(this.props.userEtherBalanceInWei, constants.DECIMAL_PLACES_ETH);
+        const isWrappingEth = this.props.direction === Side.Deposit;
+        const topLabelText = isWrappingEth ? 'Convert ETH into WETH 1:1' : 'Convert WETH into ETH 1:1';
+        return (
+            <ListItem
+                primaryText={
+                    <div>
+                        <div style={styles.topLabel}>{topLabelText}</div>
+                        <div style={styles.inputContainer}>
+                            {isWrappingEth ? (
+                                <EthAmountInput
+                                    balance={etherBalanceInEth}
+                                    amount={this.state.currentInputAmount}
+                                    hintText={'0.00'}
+                                    onChange={this._onValueChange.bind(this)}
+                                    shouldCheckBalance={true}
+                                    shouldShowIncompleteErrs={false}
+                                    shouldShowErrs={false}
+                                    shouldShowUnderline={false}
+                                    style={styles.ethAmountInput}
+                                />
+                            ) : (
+                                <TokenAmountInput
+                                    lastForceTokenStateRefetch={this.props.lastForceTokenStateRefetch}
+                                    blockchain={this.props.blockchain}
+                                    userAddress={this.props.userAddress}
+                                    networkId={this.props.networkId}
+                                    token={this.props.etherToken}
+                                    shouldShowIncompleteErrs={false}
+                                    shouldCheckBalance={true}
+                                    shouldCheckAllowance={false}
+                                    onChange={this._onValueChange.bind(this)}
+                                    amount={this.state.currentInputAmount}
+                                    hintText={'0.00'}
+                                    onVisitBalancesPageClick={_.noop}
```

Comment:
You shouldn't define this callback. Leaving it as undefined removes the balances link from the error message. As noop, it'll be there but just no do anything.

Reply:
Ah, I see you have a `shouldShowErrs` flag. In that case, I'd still remove this prop.